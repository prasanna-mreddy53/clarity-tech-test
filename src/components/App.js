import React, { Component } from "react";
import Header from "./header/header.component";
import CategoryPage from "./category-page/category-page.component";
import { Route, Switch } from "react-router-dom";

class App extends Component {
  render() {
    return (
      <div>
        <Header />
        <Switch>
          <Route exact path="/" component={CategoryPage} />
          <Route path="/categories" component={CategoryPage} />
        </Switch>
      </div>
    );
  }
}

export default App;
