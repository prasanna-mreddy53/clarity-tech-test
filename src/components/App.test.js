import React from "react";
import { shallow } from "enzyme";
import App from "./App";

const app = shallow(<App />);

describe("App Component Tests", () => {
  it("renders correctly", () => {
    expect(app).toMatchSnapshot();
  });
});
