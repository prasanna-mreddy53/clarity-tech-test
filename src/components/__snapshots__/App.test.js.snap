// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component Tests renders correctly 1`] = `ShallowWrapper {}`;
